connectors/es/document.py

```diff
@@ -46,3 +123,61 @@ async def reload(self):
         self._seq_no = doc_source.get("_seq_no")
         self._primary_term = doc_source.get("_primary_term")
         self._source = doc_source.get("_source", {})
+
+    def log_debug(self, msg, *args, **kwargs):
```

Comment:
Should the code be removed since you introduced a wrapper?

Reply:
This is syntactic sugar, so you can call `connector.log_debug` instead of `connector.logger.debug`

Reply:
It feels like it does not add much value - it just passes arguments to a different object/function, so IMO it should be inlined

Reply:
I think the value is that when you use `connector.log_debug`, you know you are logging with the context of the connector. Otherwise, you have to remember to use `connector.logger.debug`. Actually `logger` should be made `private`, i.e. `_logger`. It's not private because I have to pass it to each data source.